File: humbugg/Humbugg.Api/Models/GroupMember.cs
using MongoDB.Bson.Serialization.Attributes;
using System.Collections.Generic;

namespace Humbugg.Api.Models
{
    public class GroupMember : BaseModel
    {
        public string GroupId { get; set; }
        public string UserId { get; set; }
        public bool IsAdmin { get; set; }
        public bool IsParticipating { get; set; }
        public string RecipientId { get; set; }
        [BsonIgnore]
        public Group Group { get; set; }
        public string FirstName { get; set; }
        public string MiddleName { get; set; }
        public string LastName { get; set; }
        public string Address1 { get; set;}
        public string Address2 { get; set;}
        public string City { get; set; }
        public string State { get; set; }
        public string PostalCode { get; set; }
        public string PictureUrl { get; set; }
        public string GiftSuggestionsDescription { get; set; }
        public string GiftAvoidancesDescription { get; set; }
        public string SecretQuestionAnswer { get; set; }
    }
}

File: humbugg/Humbugg.Web/ClientApp/src/app/models/groupMember.ts
import { Group } from "../models/group";

export class GroupMember {
  id: string;
  groupId: string;
  userId : string;
  isAdmin: boolean;
  isParticipating: boolean = true;
  recipientId: string;
  group: Group;
  firstName: string;
  middleName: string;
  lastName: string;
  address1: string;
  address2: string;
  city: string;
  state: string;
  postalCode: string;
  pictureUrl: string;
  secretQuestionAnswer: string;
  giftSuggestionsDescription: string;
  giftAvoidancesDescription: string;
}
